test/losses/test_hd.py

```diff
@@ -1,7 +1,6 @@
 import numpy as np
 import pytest
 import torch
-from scipy.ndimage import convolve
```

Comment:
can you remove it from here too https://github.com/kornia/kornia/blob/90de0ffa75e17a21fe973c1f3153d4b919e3640a/requirements/requirements-dev.txt#L10

Reply:
I did it now